src/controllers/movies.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const fetch = require('node-fetch')
+
+
+const getMovies = (req, res) => {
```

Comment:
`getMoviesPage` would make more sense, reading `getMoves` I expect it to return a list of movies(array of objects)